Libplanet.Extensions.Cocona.Tests/UtilsTest.cs

```diff
@@ -24,7 +24,8 @@ public void TestHumanReadable()
 
         var serialized = Utils.SerializeHumanReadable(dummyClass);
         var dummyClass2 = Utils.DeserializeHumanReadable<DummyClass>(serialized);
-        Assert.True(dummyClass.SampleByteArray.SequenceEqual(dummyClass2.SampleByteArray));
+        Assert.NotNull(dummyClass2);
+        Assert.True(dummyClass.SampleByteArray.SequenceEqual(dummyClass2!.SampleByteArray));
```

Comment:
Is it safe if use `!` in `dummyClass2`?

Reply:
That's why I added `Assert.NotNull(dummyClass2)` right above.